run-tests.js

```diff
@@ -210,7 +210,12 @@ async function main() {
     )
   })
 
-  if ((testType && testType !== 'unit') || hasIsolatedTests) {
+  if (
+    (testType &&
+      testType !== 'unit' &&
+      !process.env.NEXT_TEST_MODE === 'deploy') ||
```

Comment:
Were you looking for `!==` instead? Otherwise, this will be comparing a boolean value to a string

Reply:
Yup typed too fast here, good catch